.github/workflows/_build-and-test-locally.yml

```diff
@@ -292,12 +292,18 @@ jobs:
       fail-fast: false
       matrix:
         pg_version: ${{ fromJson(inputs.pg-versions) }}
+        no_default_lfc: [lfc]
+        include:
+          - pg_version: 16
+            no_default_lfc: nolfc
```

Comment:
For the PR we have 3 sets of tests with disabled LFC, isn't it supposed to be just one?

Reply:
This code was removed.